build.sbt

```diff
@@ -87,7 +87,7 @@ lazy val releaseSettings = Seq(
 lazy val commonDependencies = Seq(
   "com.typesafe" % "config" % "1.4.2",
   scalatest % "test",
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
+  "com.typesafe.scala-logging" % "scala-logging_2.13" % "3.9.5",
```

Comment:
hard code to the 2.13 version because it would cause a binary incompatibility if we compile some modules against each one.  Ideally we would use the scala 3 version throughout but I thought I'd keep the existing status quo/minimal change